### ISSUE TITLE
Problems loading sample data in 2.5.4

### ISSUE THREAD

**Issue by jefffriesen** [openeemeter/eemeter#352]
Installing using `pip install eemeter` and trying to run the sample data using `eemeter.samples()` gives me the following error:
> FileNotFoundError: [Errno 2] No such file or directory: '/Users/jeff/anaconda3/envs/openee/lib/python3.7/site-packages/eemeter/samples/metadata.json'

The contents of `/Users/jeff/anaconda3/envs/openee/lib/python3.7/site-packages/eemeter/samples/` has:
```
__init__.py
__pycache__
load.py
```
No sample data or metadata.json.

However, based on issue https://github.com/openeemeter/eemeter/issues/330 I installed in a new environment using:
```
pip install -e git+git://github.com/openeemeter/eemeter.git@v2.5.4#egg=eemeter
```

`eemeter.samples()` worked at this point. 

I'm using conda to manage environments, but I suspect that isn't the problem. I would guess those assets are not being distributed in the release, but it works when cloning. Either way, here is my setup from `conda info`:

```
     active environment : openee
    active env location : /Users/jeff/anaconda3/envs/openee
            shell level : 2
       user config file : /Users/jeff/.condarc
 populated config files : /Users/jeff/.condarc
          conda version : 4.6.14
    conda-build version : 3.17.8
         python version : 3.7.3.final.0
       base environment : /Users/jeff/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/jeff/anaconda3/pkgs
                          /Users/jeff/.conda/pkgs
       envs directories : /Users/jeff/anaconda3/envs
                          /Users/jeff/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.14 requests/2.21.0 CPython/3.7.3 Darwin/18.5.0 OSX/10.14.4
                UID:GID : 501:20
             netrc file : /Users/jeff/.netrc
           offline mode : False
```

So I'm not blocked, but I thought you should know. Thanks.

**Comment by philngo** [comment]
@jefffriesen I'm going to look into these packaging issues this afternoon.

**Comment by philngo** [comment]
@jefffriesen This should be fixed. There was an issue with the MANIFEST.in file, which prevented the proper files from being included in the release. I released v2.5.4-post1 with the fix. I'll go ahead and close this and #353 since you're not blocked, but feel free to reopen or open another issue if this comes up again.

```
$ pip install eemeter==2.5.4-post1
```
These should all work now. Please be aware of the caution to use pandas < 0.24 for now (see #338).
```
>>> import eemeter
>>> eemeter.samples()
>>> eemeter.load_sample('il-electricity-cdd-hdd-daily')
```

**Comment by jefffriesen** [comment]
@philngo Thanks. That along with pandas 23.4 worked.
